### ISSUE TITLE
No output testing

### ISSUE THREAD

**Issue by MCJOHN974** [objectionary/sandbox#7]
In github jobs we just test that eo program correctly compiled and runs without failure. But we don't test that output is correct. @yegor256 if you think that this additional check is proper to this repo I added it in #6

**Comment by Graur** [comment]
@MCJOHN974 Good idea, but I think it will be better to do it in more canonical way: to add `app-test.eo` in `test` folder of `sandbox` project and test output like this:
```
[] > output-test
  assert-that > @ 
    app * 9
    $.equal-to
      "9th Fibonacci number is 34"
```

**Comment by MCJOHN974** [comment]
@Graur as described [here](https://arxiv.org/pdf/2111.13384.pdf) EO has no exceptions, but github actions need non zero return code to failure.
